fv3core/fv3core/stencils/d_sw.py

```diff
@@ -988,50 +989,47 @@ def __call__(
             cx, cy, mfx, mfy, crx, cry, self._tmp_fx, self._tmp_fy
         )
 
-        if not self.hydrostatic:
```

Comment:
The zero initialization needs to happen whether or not the model is hydrostatic. Rather than make a new stencil just to set these to zero, I removed the if-statement (we don't support non-hydrostatic) so the code doesn't have a "bug" of not setting zero if non-hydrostatic.